### PR TITLE
Adapt cp2k production

### PR DESCRIPTION
**Pull request by teojgo**

### CODE REVIEW

**Comment by teojgo** [comment]
@jenkins-cscs retry all

File: cscs-checks/apps/cp2k/cp2k_check.py [hunk 1]
```diff
@@ -50,10 +50,10 @@ def __init__(self, **kwargs):
         self.num_tasks_per_node = 36
         self.reference = {
             'dom:mc': {
-                'perf': (340, None, 0.15)
+                'perf': (174.52, None, 0.05)
```
Comment (victorusu) [inline cscs-checks/apps/cp2k/cp2k_check.py]: I would round to `174.5` so it is easier to parse. What do you think?

File: cscs-checks/apps/cp2k/cp2k_check.py [hunk 2]
```diff
@@ -50,10 +50,10 @@ def __init__(self, **kwargs):
...
             },
             'daint:mc': {
-                'perf': (113, None, 0.15)   # 'perf': (186, None, 0.15)
+                'perf': (113.89, None, 0.25)
```
Comment (victorusu) [inline cscs-checks/apps/cp2k/cp2k_check.py]: I would leave as `113.0`, what do you think?

File: cscs-checks/apps/cp2k/cp2k_check.py [hunk 3]
```diff
@@ -95,10 +95,10 @@ def __init__(self, **kwargs):
         self.tags |= {'production'}
         self.reference = {
             'dom:gpu': {
-                'perf': (258, None, 0.15)
+                'perf': (239.27, None, 0.05)
```
Comment (victorusu) [inline cscs-checks/apps/cp2k/cp2k_check.py]: I would round to `240.0` so it is easier to parse. What do you think?

File: cscs-checks/apps/cp2k/cp2k_check.py [hunk 4]
```diff
@@ -95,10 +95,10 @@ def __init__(self, **kwargs):
...
             },
             'daint:gpu': {
-                'perf': (195, None, 0.15)
+                'perf': (195.38, None, 0.10)
```
Comment (victorusu) [inline cscs-checks/apps/cp2k/cp2k_check.py]: I would leave it as `195.0` so it is easier to parse. What do you think?

**Pull request merged**